.github/scripts/parse_rpt.py

```diff
@@ -102,11 +105,48 @@
 
 ## Result File Check
 if _generator_is['sky130hvl_ldo']:
-   json_filename = "spec.json"
+    json_filename = "spec.json"
 else:
-   json_filename = "test.json"
+    json_filename = "test.json"
 
 if check_gen_files(json_filename, _generator_is, cryo_library):
-        print("Flow check is clean!")
+    print("Flow check is clean!")
 else:
     print("Flow check failed!")
+
+if len(sys.argv) == 1:
+    sim_state_filename = "work/sim_state_file.txt"
+
+    if check_ngspice_version():
+        result_filename = "work/prePEX_sim_result" 
+
+        template_filename = "../../../.github/scripts/expected_sim_outputs/prePEX_sim_result.txt"
+        with open(result_filename) as f2, open(template_filename) as f1:
+            content1 = f2.readlines()
+            content2 = f1.readlines()
+            if content1 != content2:
```

Comment:
Will the content be exactly the same each time? The numbers are very precise and may probably vary slightly in each run?

Reply:
Yeah they might vary, that's why it isnt a hard requirement for the script to succeed

Reply:
What is the hard requirement?

Reply:
the number of failed simulations should be 0 (returned in the dictionary at the end of simulation flow)